flow/connectors/snowflake/qrep.go

```diff
@@ -296,10 +301,10 @@ func (c *SnowflakeConnector) getColsFromTable(tableName string) (*model.ColumnIn
 	}
 	defer rows.Close()
 
+	var colName pgtype.Text
```

Comment:
why were these moved outside the loop?

Reply:
I'd guess the goal is to avoid clearing an empty stack slot each loop, which may also be boxed depending how go analyzes the Scan. This refactor has been appearing in a few previous PRs